functions/__bobthefish_glyphs.fish

```diff
@@ -70,6 +71,7 @@ function __bobthefish_glyphs -S -d 'Define glyphs used by bobthefish'
     set nix_glyph        \uF313 ' ' # nf-linux-nixos
     set virtualenv_glyph \uE73C ' '
     set ruby_glyph       \uE791 ' '
+    set rustc_glyph       \uE7a8 ' ' # Rust Nerd Font logo
```

Comment:
Minor things but you clearly copied the Ruby line, which has one fewer character, for these two and the white space is off. Do you mind lining them all up? :)